drivers/display/display_ili9340.c

```diff
@@ -254,10 +276,9 @@ static void ili9340_get_capabilities(const struct device *dev,
 	capabilities->supported_pixel_formats = PIXEL_FORMAT_RGB_565 |
 						PIXEL_FORMAT_RGB_666;
 	capabilities->current_pixel_format = data->pixel_format;
-
 	capabilities->x_resolution = ILI9340_X_RES;
```

Comment:
X should represent horizontal resolution and y vertical

Reply:
Current ILI9340 driver resolution is swapped w.r.t specs. Without rotation display should report 240 (W) x 320 (H).

![image](https://user-images.githubusercontent.com/25011557/93668734-da1f3600-fa8e-11ea-8c1c-a2765999a513.png)

Reply:
fine for me, is in line with the spec